packages/ember-routing-htmlbars/lib/keywords/closure-action.js

```diff
@@ -67,6 +67,22 @@ export default function closureAction(morph, env, scope, params, hash, template,
   return s;
 }
 
+
+function extractRawAction(params, scope) {
+  var [value] = params;
+  if (value) {
```

Comment:
From what I can remember (it has been a while :) ), `closureAction()` can be passed either a stream _or_ a POJO.  In the event that it gets a stream that references a falsy action, the error thrown is descriptive enough to aid debugging.  In the event it throws a POJO, it's completely cryptic (ala `Uncaught TypeError: Cannot read property 'INVOKE [id=__ember14526275715741349532282241]' of undefined`).

I think the determinant for whether this function gets passed is whether or not you reference something off of `attrs`, e.g., `attrs.someFalsyAction`.